src/Pages/Categories.tsx

```diff
@@ -140,54 +140,52 @@ export default function Categories() {
         // "map_thumb_10.png"
     );
     const mapImgem = useProgressiveImage(
-        "embedded.png",
+        "aaa.png",
```

Comment:
Tu ta zmiana nazwy zgodna ze zdjęciem

Reply:
tak xdd, no bo tak wrzucone zeby sie nie rozjezdzalo

Reply:
Ale to zmienić musisz :D Bo aaa.png nic nie mówi co to za zdjęcie.